hack/util.sh

```diff
@@ -641,7 +641,11 @@ function install_registry {
 }
 
 function wait_for_registry {
-	wait_for_command '[[ "$(oc get endpoints docker-registry --output-version=v1 --template="{{ if .subsets }}{{ len .subsets }}{{ else }}0{{ end }}" --config=${ADMIN_KUBECONFIG} || echo "0")" != "0" ]]' $((5*TIME_MIN))
+	local config="--config=${ADMIN_KUBECONFIG}"
+	local tmplsubsets='{{ if .subsets }}{{ len .subsets }}{{ else }}0{{ end }}'
+	wait_for_command '[[ "$(oc get endpoints docker-registry --output-version=v1 --template='"'$tmplsubsets' $config"' || echo 0)" != "0" ]]' $((5*TIME_MIN))
```

Comment:
`wait_for_command` is pretty much deprecated. We should use `os::cmd::wait_until_text`:

```
os::cmd::wait_until_text "oc get endpoints docker-registry --template='{{ len .subsets }} '" "[1-9][0-9]*" "$((5*TIME_MIN))"
```